suricata.yaml.in

```diff
@@ -293,6 +293,7 @@ outputs:
         - rfb
         - sip
         - quic
+        - arp
```

Comment:
I'm wondering if we should disable this by default, it produces quite a bit of entries.

Reply:
For an attacker to overwhelm a sensor they would need to be on the same subnet? Or could it be encapsulated and then pushed over IP?

Reply:
Same subnet, or some VPN/encapsulation that bridged ethernet I guess.  Usually not "public" facing stuff.

Reply:
I can send you ARP over GRE, and suri would pick it up I think?
```python
#!/usr/bin/env python
from scapy.all import *

pkts = []

pkts += Ether(dst='05:04:03:02:01:00', src='00:01:02:03:04:05')/Dot1Q(vlan=6)/IP(src='1.1.1.1', dst='2.2.2.2')/GRE()/ARP()

wrpcap('arp-encap.pcap', pkts)
```

Reply:
Yeah, good point. Depending on where you are running Suri, you could be open to flooding.

Reply:
I guess a question is: should we make the decoder configurable wrt encapsulation? E.g. by default only allow over ethernet directly?

Reply:
I agree with having it disabled by default.

Reply:
@glongo any thoughts on allowing it only for direct ethernet vs encapsulations?

Reply:
I would decode ARP packets in both scenarios, direct ethernet and encapsulated, without making the decoder configurable.